docs/how-to/build-interacting-with-the-network.md

```diff
@@ -28,8 +28,13 @@ The same containerization scripts for [local networks](#local-networks) can also
 
 TODO: Additional tips.
 
-## Testnets / Mainnet
+## Testnet
 
 [Filecoin Testnet](https://filecoin.io/testnet/) is a live community test network, under active development.
 
+### Hosted nodes for Testnet
+The Filecoin project has a limited number of dedicated, up-to-date nodes (synced with the latest Testnet) available to developers actively building storage services or applications. They are intended to jump-start development, not for production usage. To request a dedicated hosted node, please email [filecoin-collabs@protocol.ai](mailto:filecoin-collabs@protocol.ai?subject=Requesting20%a20%hosted20%node).
```

Comment:
Synced with the latest Testnet _chain_? 

Does everyone on Testnet use hosted nodes or do most host their own? If I skimmed through this it might feel like hosted nodes were the way to use Testnet since nothing else is mentioned in the section.

Reply:
chain sounds right, will fix.

Most people on testnet host their own. I'll add a clarification.